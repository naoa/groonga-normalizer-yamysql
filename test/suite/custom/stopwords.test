register normalizers/yamysql

table_create @yamysql_stopwords TABLE_HASH_KEY ShortText
load --table @yamysql_stopwords
[
{"_key": "テスト"}
]
normalize NormalizerYaMySQL "これはテストです。" WITH_TYPES|WITH_CHECKS

