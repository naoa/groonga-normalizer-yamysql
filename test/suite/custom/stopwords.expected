register normalizers/yamysql
[[0,0.0,0.0],true]
table_create @yamysql_stopwords TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table @yamysql_stopwords
[
{"_key": "テスト"}
]
[[0,0.0,0.0],1]
normalize NormalizerYaMySQL "これはテストです。" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "これは   です。",
    "types": [
      "hiragana",
      "hiragana",
      "hiragana",
      "unknown",
      "unknown",
      "unknown",
      "hiragana",
      "hiragana",
      "symbol"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      3,
      3,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
